features/action_officer_spec.rb

```diff
@@ -61,4 +61,24 @@ def create_ao(name, email)
     expect(page).to have_content 'an action officer cannot be assigned twice to the same deputy director'
     expect(page).not_to have_content 'Action officer was successfully created'
   end
+
+  scenario 'Parli-branch can see inactive action officers' do
+    create_pq_session
+    visit action_officers_path
+    ao = ActionOfficer.last
+    expect(page).to have_content ao.name
+
+    ao.update_attribute :deleted, true
+    ao.update_attribute :updated_at, 2.weeks.ago
+    visit action_officers_path
+    expect(page).not_to have_content ao.name
+
+    click_link 'View inactive action officers'
+    expect(page).to have_content ao.name
+    expect(page).to_not have_content 'View inactive action officers'
```

Comment:
I just spotted this, never came across the `to_not` versus `not_to` thing before - they both work but I think using one consistently would be good 

Hard to find reasons one way or the other but the [rubocop for rspec prefers not_to](https://www.rubydoc.info/gems/rubocop-rspec/1.7.0/RuboCop/Cop/RSpec/NotToNot) so can we change the second one?

```suggestion
    expect(page).not_to have_content 'View active action officers'
```